qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -51,7 +63,7 @@ wait_for_jobs() {
         # Get the list of all jobs in the helm release, and subtract the value of 'completed' from 'desired'
         # It would be better to parse this from `helm get manifest`, but since that command is broken in helm
         # v2.8.2 (https://github.com/helm/helm/issues/3833) for now we'll parse it from the human-readable status
-        jobs_desired_remaining=$(helm status $release | awk '/==> v1\/Job/ { getline; getline; while (NF>0) { print $2 - $3; getline } }')
+        jobs_desired_remaining=$(helm${helm_ver} status $release | awk '/==> v1\/Job/ { getline; getline; while (NF>0) { print $2 - $3; getline } }')
```

Comment:
Can we assume `helm_ver` will *always* be set here?

Reply:
yes. its set in `/lib/prepare-kubeconfig.sh`

Reply:
resolved now with changes to helm cli invocation.